Makefile

```diff
@@ -1,12 +1,25 @@
 PREFIX ?= /usr
 DESTDIR ?=
 
-.PHONY: all mantle install
+.PHONY: all mantle install check clean
 
 all: mantle
 
-check:
-	./tests/check.sh
+src:=$(shell find src -maxdepth 1 -type f -executable -print)
+src_checked:=$(patsubst src/%,src/.%.shellchecked,${src})
+tests:=$(shell find tests -maxdepth 1 -type f -executable -print)
+tests_checked:=$(patsubst tests/%,tests/.%.shellchecked,${tests})
+cwd:=$(shell find . -maxdepth 1 -type f -executable -print)
+cwd_checked:=$(patsubst ./%,.%.shellchecked,${cwd})
+
+.%.shellchecked: %
+	./tests/check_one.sh $< $@
```

Comment:
the `%` after the target, target name

Reply:
I don't understand?

Reply:
This was meant as clarification for a question that came up in chat.